pom.xml

```diff
@@ -3,7 +3,7 @@
 
     DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 
-    Copyright (c) 2011-2015 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2011-2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
2011-2015 refers to Oracle versioning
Suggestion: add a comment bellow with 2023 regarding Payara versioning

Reply:
Fixed, thank you!